cmd/solution/push.go

```diff
@@ -40,10 +40,9 @@ var solutionPushCmd = &cobra.Command{
 	Long: `This command allows the current tenant specified in the profile to deploy a solution to the FSO Platform.
 The solution manifest for the solution must be in the current directory.`,
```

Comment:
Add more details for tags here:
1. tags is a free-form flag which means you can add any string there however 'stable' is a reserved key-word for production-ready bundles.
2. Use caution while typing tag values between validate and upload commands since typos can result in misleading validation results.
3. For more detailed info on tags goto -> user docs link.